app/assets/stylesheets/components/comments.scss

```diff
@@ -45,10 +53,6 @@
       .c-embed__body {
         font-size: var(--fs-s);
       }
-      div:not(.crayons-comment__readmore) {
-        margin-bottom: var(--su-3);
-        background: var(--card-secondary-bg);
-      }
```

Comment:
I have removed this to fix the design issue shown below. Are there any other side-effects of this change?

| Before  | After |
| ------------- | ------------- |
| <img width="587" alt="Screenshot 2023-05-08 at 11 28 05 AM" src="https://user-images.githubusercontent.com/9396084/236748036-17d1f340-1552-452f-b9fc-8b7e03a4d20e.png"> | <img width="549" alt="Screenshot 2023-05-08 at 11 28 28 AM" src="https://user-images.githubusercontent.com/9396084/236748060-2e76a925-6f3e-48bb-b458-a8976315d46f.png"> |

Reply:
Yeah, that is actually necessary to get liquid embeds to show up properly. Since these divs have all sorts of different classes, I just used `div`, but as this issue points out, this is a problem.

I think for the sake of this fix, you might need to instead keep it, but chain the extra class on, so:

```
div:not(.crayons-comment__readmore):not(.highlight) {
...
}
```

Not the most beautiful situation, but for now probably gets us to the best overall place.

Reply:
Fixed this by applying suggested change.

<img width="669" alt="Screenshot 2023-05-08 at 7 28 58 PM" src="https://user-images.githubusercontent.com/9396084/236844196-8b1dc609-9357-4f56-ae1f-8ad04dfa95c6.png">